backend/ethereum/wallet/address.go

```diff
@@ -44,8 +46,17 @@ func (a *Address) MarshalBinary() ([]byte, error) {
 // Decode decodes an address from a io.Reader. Part of the
 // go-perun/pkg/io.Serializer interface.
 func (a *Address) Decode(r io.Reader) error {
-	buf := make([]byte, common.AddressLength)
-	_, err := io.ReadFull(r, buf)
+	var length int64
```

Comment:
Updated to use uint16 in b6137b1cdfaa284311fdb26a48d38c0ce24f1508. However, this commit will not be updated. Because, the decoding part is moved to `pkg/io` when introducing BinaryUnmarshaler in the later commit.